mapclassify/util.py

```diff
@@ -0,0 +1,71 @@
+from ._classify_API import classify
+
+
+def get_rgba(
+    values,
+    classifier="quantiles",
+    cmap="viridis",
+    alpha=1,
+    nan_color=[255, 255, 255, 255],
+    **kwargs
+):
+    """Convert array of values into RGBA colors using a colormap and classifier.
+
+    Parameters
```

Comment:
kwargs should probably be mentioned here.

Reply:
cool, i'm always unsure what to do in the docstrings when there's a catchall

Reply:
I always mention where are they passed to.